src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptor.cs

```diff
@@ -6,6 +6,8 @@
 
 namespace Microsoft.AspNet.Razor.TagHelpers
 {
+    // All properties on this class have a setter to enable serialization/deserialization.
```

Comment:
nit: comments like this go between doc comments and the class declaration.  main sometimes-valid case for whitespace after a comment is a "header" for a block of related code that's all at the same level e.g. https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc/MvcServices.cs#L42.  just harder to read this way.